label_studio/data_manager/actions/basic.py

```diff
@@ -25,9 +28,11 @@ def delete_tasks(project, queryset, **kwargs):
     :param queryset: filtered tasks db queryset
     """
     count = queryset.count()
+    tasks_ids = list(queryset.values('id'))
```

Comment:
Why not `.values_list()` ?

Reply:
`.values_list()` returns a tuple, and it can't be serialized by DRF serializer. `values()` returns a dict and it can be.